### PR TITLE
Add integration testing example

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: integration-testing/main_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+
+			client, err := spicedbTestClient(t, port)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			_, err = client.WriteSchema(context.TODO(), &v1.WriteSchemaRequest{Schema: tt.schema})
```
Comment (josephschorr) [inline integration-testing/main_test.go:94]: We should show an example where the schema + test data is preloaded into serve-testing

Comment (jzelinskie) [reply integration-testing/main_test.go:94]: Sounds good for a follow-up. I just wanted to get something down for now.

File: integration-testing/main_test.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tests := []struct {
+		name   string
+		schema string
+	}{
+		{
+			"basic readback",
```
Comment (josephschorr) [inline integration-testing/main_test.go:80]: Add at least one more test here

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**